components/presenters/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react'
+import { Badge } from '@royalnavy/react-component-library'
+import { IconChevronLeft } from '@royalnavy/icon-library'
+
+import { ComponentWithClass } from '../../../common/ComponentWithClass'
+import { StyledFrame } from './partials/StyledFrame'
+import { StyledSidebar } from './partials/StyledSidebar'
+import { StyledHomeLink } from './partials/StyledHomeLink'
+import { StyledHeader } from './partials/StyledHeader'
+import { StyledTitle } from './partials/StyledTitle'
+import { StyledButton } from './partials/StyledButton'
+
+interface SidebarProps extends ComponentWithClass {
+  title: string
+}
+
+export const Sidebar: React.FC<SidebarProps> = ({ title, children }) => {
+  const [menuToggle, setMenuToggle] = useState(false)
+
+  return (
+    <StyledFrame $isOpen={menuToggle}>
+      <StyledSidebar>
+        <div>
+          <StyledHomeLink href="/">
+            <IconChevronLeft />
+            Back to docs
+          </StyledHomeLink>
+          <StyledHeader>
+            <Badge color="action" colorVariant="solid" size="small">
+              Framework
+            </Badge>
+            <StyledTitle data-testid="sidebar-title">{title}</StyledTitle>
+          </StyledHeader>
+          <StyledButton
+            type="button"
+            onClick={() => setMenuToggle(!menuToggle)}
```

Comment:
All the tests pass if this line is removed.

Reply:
This internal state is kind of placeholder it should invoke a callback really. 

I was going to change this when we started stitching things together but I'll just update it now 👍